packaging/debian/dotnet-debian_config.json

```diff
@@ -33,13 +33,17 @@
         "libcurl3" : {}
     },
 
+    "package_conflicts" : [
+        "dotnet-nightly"
```

Comment:
I think we're going with the `-dev` moniker. It seems more consistent. @krwq for details.